internal/server/webhooks/server.go

```diff
@@ -36,7 +36,8 @@ func StartWebhooks(config string) {
 
 	err = json.Unmarshal(activeWebhooks, &recipients)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return
```

Comment:
Why?

Reply:
webhook is not a critical feature, it should allow failure

Reply:
That isnt what this code does. While agree webhooks arent a critical feature, if someone has a configuration file that indicates they *want* webhooks, but we cant parse it, thats an error that is critical and they need to sort it out.

Reply:
okie ,fixed it ,just wait a new pr